i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments/deploy-on-kubernetes/common-helm-chart-configurations.md

```diff
@@ -0,0 +1,302 @@
+
+用户可使用如下字段来配置服务端口，如下所示：
+
+- `httpServicePort`： 用于配置 HTTP 服务的端口，默认 4000；
+- `grpcServicePort`： 用于配置 SQL 服务的端口，默认 4001；
+- `mysqlServicePort`： 用于配置 MySQL 服务的端口，默认 4002；
+- `postgresServicePort`： 用于配置 PostgreSQL 服务的端口，默认 4003；
+
+### Datanode 存储配置
+
+用户可以通过 `datanode.storage` 字段来配置 Datanode 的存储，如下所示：
+
+```yaml
+datanode:
+  storage:
+    # -- Storage class for datanode persistent volume
+    storageClassName: null
+    # -- Storage size for datanode persistent volume
+    storageSize: 10Gi
+    # -- Storage retain policy for datanode persistent volume
+    storageRetainPolicy: Retain
+    # -- The dataHome directory, default is "/data/greptimedb/"
+    dataHome: "/data/greptimedb"
+```
+
+- `storageClassName`： 用于配置 StorageClass，默认使用 Kubernetes 当前默认的 StorageClass；
+- `storageSize`： 用于配置 Storage 的大小，默认 10Gi；
+- `storageRetainPolicy`： 用于配置 Storage 的保留策略，默认 `Retain`，如果设置为 `Delete`，则当集群被删除时，相应的 Storage 也会被删除；
+- `dataHome`： 用于配置数据目录，默认 `/data/greptimedb/`；
+
+### 运行资源配置
+
+顶层变量 `base.podTemplate.main.resources` 用于全局配置每个 Role 的资源，如下所示：
+
+```yaml
+base:
+  podTemplate:
+    main:
+      resources:
+        requests:
+          memory: "1Gi"
+          cpu: "1"
+        limits:
+          memory: "2Gi"
+          cpu: "2"
+```
+
+如果你想为集群中的每个 Role 配置不同的资源，可以使用 `${role}.podTemplate.main.resources` 字段（其中 `role` 可以是 `meta`、`frontend`、`datanode` 等），改字段会**覆盖顶层**变量 `base.podTemplate.main.resources` 的配置，如下所示：
+
+```yaml
+base:
+  podTemplate:
+    main:
+      resources:
+        requests:
+          memory: "1Gi"
+          cpu: "1"
+        limits:
+          memory: "2Gi"
+          cpu: "2"
+
+frontend:
+  podTemplate:
+    main:
+      resources:
+        requests:
+          cpu: "2"
+          memory: "4Gi"
+        limits:
+          cpu: "4"
+          memory: "8Gi"
+```
+
+### 服务运行副本数配置
+
+对于不同 Role 的副本数，可以通过 `${role}.replicas` 字段进行配置对应的副本数，如下所示：
+
+```yaml
+frontend:
+  replicas: 3
+
+datanode:
+  replicas: 3
+```
+
+用户可以通过配置其副本数来实现水平扩缩。
+
+### 环境变量配置
+
+我们既可以通过 `base.podTemplate.main.env` 字段配置全局的环境变量，也可以通过 `${role}.podTemplate.main.env` 字段为每个 Role 配置不同的环境变量，如下所示：
+
+```yaml
+base:
+  podTemplate:
+    main:
+      env:
+        - name: GLOBAL_ENV
+          value: "global_value"
+
+frontend:
+  podTemplate:
+    main:
+      env:
+        - name: FRONTEND_ENV
+          value: "frontend_value"
+```
+
+### 注入配置文件
+
+对于不同 Role 的服务，我们可以通过 `${role}.configData` 字段注入自定义的 TOML 配置文件，如下所示：
+
+```yaml
+frontend:
+  configData: |
+    [[region_engine]]
+    [region_engine.mito]
+    # Number of region workers
+    num_workers = 8
+```
+
+用户可以通过 [config.md](https://github.com/GreptimeTeam/greptimedb/blob/main/config/config.md) 了解 GreptimeDB 的配置项。
+
+除了使用 `${role}.configData` 字段注入配置文件，用户还可以通过 `${role}.configFile` 来指定相应的文件，如下所示：
+
+```yaml
+frontend:
+  configFile: "configs/frontend.toml"
+```
+
+此时需要确保对应的配置文件路径与执行 `helm upgrade` 命令时所处的目录一致。
+:::note
+用户注入的配置文件默认优先级低于由 GreptimeDB Operator 所接管的配置项，某些配置项仅能通过 GreptimeDB Operator 进行配置，而这些配置项默认会暴露在 `values.yaml` 中。
```

Comment:
It would be better to clearly write out which items in "values.yaml" changing will have no effect.